german/ruderboot.md

```diff
@@ -9,3 +9,5 @@ Werden Sie:
 [Laut um Hilfe schreien](Hilfeschrei/Hilfeschrei.md)?
 
 [Gefasst bleiben und ihre Kleidung ausziehen und aus dem Boot werfen?](gefasst-bleiben/gefasst-bleiben.md)
+
+[Obligatory link to new file](test-dir/K�lte.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.